apis/memcached/v1beta1/memcached_types.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	condition "github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MemcachedSpec defines the desired state of Memcached
+type MemcachedSpec struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="quay.io/tripleozedcentos9/openstack-memcached:current-tripleo"
+	// Name of the memcached container image to run
+	ContainerImage string `json:"containerImage,omitempty"`
+
+	// +kubebuilder:default=1
```

Comment:
I could make it option and rely on a default value. Is that pattern already implemented in another operator that you know of? Otherwise I'll propose something

Reply:
This is the default value. so you just have to add the optional tag, like in the above `ContainerImage`.

Reply:
Ah sorry I didn't get your point. Fixed now